lib/tic_tac_toe_checker.ex

```diff
@@ -66,7 +67,7 @@ defmodule TicTacToeChecker do
 
   @impl GenServer
   def handle_call(:state, _from, state) do
-    {:reply, state, Map.put(state, :reported, true)}
+    {:stop, :normal, state, Map.put(state, :reported, true)}
```

Comment:
Eh. Why querying the state should result in the process exit? We need to either rename the `call` event from `:state` to smth else or not stop the process.